pydataproc/dataproc.py

```diff
@@ -185,15 +185,46 @@ def delete_cluster(self, cluster_name):
             clusterName=cluster_name).execute()
         return result
 
-    # TODO improve job submission - make it easier to specify jobs
-    # without needing to create a large job_details dict
-    def submit_job(self, job_details):
-        """
-        Submit a job to a cluster.
-
-        :param job_details: A dict describing the job to be submitted
+    def submit_job(self, cluster_name, file_to_run, python_files=None, args=[]):
```

Comment:
The list of args seems clunky. `[key, val, key, val]` as a list doesn't really give you much over a simple string `key val key val`.split().

Reply:
Ah, good catch, that's a lot cleaner 👍